main/rxvt-unicode/APKBUILD

```diff
@@ -57,13 +58,23 @@ build() {
 }
 
 package() {
+	# despite having a separate terminfo subpackage
+	# TERMINFO env var is used by rxvt-unicode makefile
+	# leaving it as is ~skrzyp
 	export TERMINFO="$pkgdir/usr/share/terminfo"
 	install -d "$TERMINFO"
 
 	make DESTDIR="$pkgdir" \
 		-C "$_builddir" install || return 1
 }
 
+terminfo() {
```

Comment:
Please use only tabs for indentation (and alignment) in APKBUILDs and other shell scripts, not spaces.